tssc/step_implementers/unit_test/maven.py

```diff
@@ -115,6 +122,22 @@ def required_runtime_step_config_keys():
         """
         return REQUIRED_CONFIG_KEYS
 
+    def _settings_file(self):
```

Comment:
@ppremru @akim309 suggest calling this `__generate_maven_settings` for consistancy.